packages/coding-standards/README.md

```diff
@@ -31,9 +31,9 @@ and even add your own sniffs and fixers in the config.
 
 ```yaml
 #custom-coding-standard.yml
-includes:imports:
- - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
- - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
+imports:
+   - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
+   - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
```

Comment:
It's probably out of scope of this PR, but `psr2.yml` should be in `shopsys-coding-standard.yml`.

In the end, people load only one config (the Shopsys one).

See Sylius Coding Standard for inspiration: https://github.com/SyliusLabs/CodingStandard#installation--usage